lib/spaceship/tunes/tunes_client.rb

```diff
@@ -208,5 +208,92 @@ def build_trains(app_id)
       data = parse_response(r, 'data')
     end
 
+    #####################################################
+    # @!group Testers
+    #####################################################
+    def testers
+      r = request(:get, "ra/users/pre/ext")
+      parse_response(r, 'data')['testers']
+    end
+
+    def testers_by_app(app_id) 
+      r = request(:get, "ra/user/externalTesters/#{app_id}/")
+      parse_response(r, 'data')['users']
+    end
+
+    def create_tester!(email: nil, first_name: nil, last_name: nil) 
+      # TODO: Get struct from the api
+      data = {
+        testers: [
+          {
+            email_address: {
+              value: email
+            }, 
+            first_name: {
+              value: first_name
+            },
+            last_name: {
+              value: last_name
+            },
+            testing: {
+              value: true
+            }
+          }
+        ]
+      }
+
+      # Now send back the modified hash
+      r = request(:post) do |req|
+        req.url 'ra/users/pre/create'
+        req.body = data.to_json
+        req.headers['Content-Type'] = 'application/json'
+      end
+        
+      data = parse_response(r, 'data')
+      handle_itc_response(data)
+    end
+
+    def remove_tester_from_app(tester, app_id)
+      # TODO: Get struct from the api
+      data = {
+        users: [
+          {
+            email_address: {
+              value: tester.email,
+              isEditable: true,
+              isRequired: true,
+              errorKeys: nil
+            }, 
+            first_name: {
+              value: tester.first_name,
+              isEditable: true,
+              isRequired: false,
+              errorKeys: nil
+            },
+            last_name: {
+              value: tester.last_name,
+              isEditable: true,
+              isRequired: false,
+              errorKeys: nil
+            },
+            testing: {
+              value: false,
+              isEditable: true,
```

Comment:
You're right, I added this attributes because I'm receiving an error in the Post but it didn't solve the problem, I will see it. Thanks!